### DISCUSSION TITLE
Question: Authentication with Firebase

### DISCUSSION THREAD

**Discussion by camstuart** [elimintz/justpy#690] [Q&A]
Hello,

is there any examples or documentation that we could use to implement authentication with Firebase?

Thanks

**Comment by WolfgangFahl** [comment]
Not that i know of.
I am continuously frustrated that questions like this are opened as issues and the [if you have questions-suggestions-or would like to get in touch](https://justpy.io/#if-you-have-questions-suggestions-or-would-like-to-get-in-touch) is completly ignored.
